### DISCUSSION TITLE
Some Legal Enquiry

### DISCUSSION THREAD

**Discussion by Wjplatformer** [TomSchimansky/CustomTkinter#615] [Q&A] [answered]
So I am currently building an app to launch to the public, using this as one of the dependencies. Could I trademark the app or something? It's called Hot-Web btw, and uses this as well the main UI. It's going to be Open Sourced on GitHub in the distant-ish future and will not contain source code of CustomTkinter. Just wondering if I can publish it. 🥺

**Comment by TomSchimansky** [comment, marked as answer]
The license is 'Creative Commons Zero v1.0 Universal', so there are no limitations at all. You can do whatever you want with the library, you can also use it commercially without mentioning the owner. But of course you are welcome to link this Github repository somehow in your project or on the Github page of your project. I would be interested to see the finished project, maybe you can post a link here if it's published.

**Reply by Wjplatformer** [reply]
Oh thanks so much! :)

**Reply by Wjplatformer** [reply]
I would probably publish it sometime next year, as it is still just starting
